components/IscnRegisterForm.vue

```diff
@@ -874,6 +877,16 @@ export default class IscnRegisterForm extends Vue {
   }
 
   async calculateISCNFee(): Promise<void> {
+    try {
+      const [ data ] = await Promise.all(
+        this.likerIds.map((e) =>
+          this.$axios.get(getLikerIdMinApi(e as string),
+        )),
+      )
+      this.likerIdsAddress = data?.data?.likeWallet
+    } catch {
```

Comment:
Please always add `console.error()` unless only an expected error would occur
Event if expected to be no-op, a console log is helpful if some unexpected error happen